scenarioo-client/test/e2e/specs/configure_label_colors.ts

```diff
@@ -46,4 +45,25 @@ useCase('Configure label colors')
 
             });
 
+        scenario('Create a label configuration with a custom color')
+            .it(async () => {
+                const customLabelColor: string = '#00FF00';
+                const numberOfPreconfiguredColors: number = 3;
+
+                await labelConfigurationsPage.navigateToPage();
+                await step('show label configurations');
+
+                await labelConfigurationsPage.addLabelConfigurationWithCustomColor('added-label', customLabelColor);
+                await step('add label configuration with a custom color');
+
+                await HomePage.goToPage();
+                await step('navigate away from the label config page to some other page');
+
+                await labelConfigurationsPage.navigateToPage();
+                await labelConfigurationsPage.assertNumConfigurations(numberOfPreconfiguredColors + 1);
+                await labelConfigurationsPage.assertConfigurationColor(3, customLabelColor);
+                await step('go back to label config page, label is still there and has the custom color');
+
+            });
```

Comment:
random color button wurde nicht geprüft

Reply:
Added a check for the random button in the e2e tests.